internal/tofu/context_apply_test.go

```diff
@@ -11410,7 +11410,7 @@ locals {
 // Ensure that we can destroy when a provider references a resource that will
 // also be destroyed
 func TestContext2Apply_destroyProviderReference(t *testing.T) {
-	m, snap := testModuleWithSnapshot(t, "apply-destroy-provisider-refs")
+	m, snap := testModuleWithSnapshot(t, "apply-destroy-provider-refs")
```

Comment:
This corresponds to a directory that a test runs in and will need to be renamed.